### PR TITLE
remove enums support in Flutter/Dart

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

**Review by PineappleIOnic — APPROVED** [review]

File: templates/flutter/lib/enums.dart.twig [hunk 1]
```diff
@@ -7,11 +7,3 @@ extension HttpMethodString on HttpMethod {
     return this.toString().split('.').last.toUpperCase();
   }
 }
```
Comment (eldadfux) [inline templates/flutter/lib/enums.dart.twig:9]: Don't we need to completely remove this template file?

File: templates/dart/lib/enums.dart.twig [hunk 2]
```diff
@@ -7,11 +7,3 @@ extension HttpMethodString on HttpMethod {
     return this.toString().split('.').last.toUpperCase();
   }
 }
```
Comment (eldadfux) [inline templates/dart/lib/enums.dart.twig:9]: Don't we need to completely remove this template file?

File: templates/flutter/lib/enums.dart.twig (hunk 1 above)
Comment (lohanidamodar) [reply templates/flutter/lib/enums.dart.twig:9]: this HttpMethod is used internally, so required.

File: templates/dart/lib/enums.dart.twig (hunk 2 above)
Comment (lohanidamodar) [reply templates/dart/lib/enums.dart.twig:9]: Same, HttpMethod emum is used internally and required

**Pull request merged**